config/default/manager_auth_proxy_patch.yaml

```diff
@@ -51,8 +51,13 @@ spec:
         args:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
+        - "--pprof-bind-address=:8082"
```

Comment:
I do not think that it is necessary to explicitly provide a setting that is the same as the default. Also, not particularly harmful.

Reply:
To be honest, I could not figure out why you kept talking about port 8082 when it was nowhere to be seen. Only after your run the help command showing the flag and port it was clear to me.
I think clearly stating the port bring more clarity.
Otherwise, one in the future may wander why open a container port 8082 that does not seem used.